homeassistant/loader.py

```diff
@@ -146,6 +146,22 @@ async def async_get_config_flows(hass: "HomeAssistant") -> Set[str]:
     return flows
 
 
+async def async_preload_integrations_with_config_flows(hass: "HomeAssistant") -> None:
+    """Preload integrations with config flows.
+
+    This function should not load the integration in
+    parallel.
+
+    Its current use case is to load each one in the background
```

Comment:
We will only load these translations when the user is on the integrations screen in the config panel, or has been there and kept the UI open. 

Wouldn't it make more sense to use `gather_with_concurrency` for translations too?

Reply:
Since translations are cached, it's only going to be slow the first time we load config flow translations after each reboot.

Reply:
Right, the case I'm solving for is where we end up with the websocket connecting while the system is still starting and we have two rounds of `async_get_integrations` 

I actually can't get that to happen anymore after https://github.com/home-assistant/core/pull/43085 😄 so I'll close this.